src/hiopbbpy/problems/problem.py

```diff
@@ -5,19 +5,19 @@
             Nai-Yuan Chiang <chiang7@llnl.gov>
 """
 import numpy as np
+import collections.abc
```

Comment:
is `collections` a new dependency?

Reply:
Collections is a built in module that is not pip installable. See [here](https://stackoverflow.com/questions/45738414/how-to-install-collections-in-python-3-6).